### ISSUE TITLE
Alphafold exits without result, but with warning of maximum number of residues 32763 exceeded in sequence

### ISSUE THREAD

**Issue by qingsongli1979** [deepmind/alphafold#135]
Alphafold exists without result, but reamins lots of warnings such as below:

I0818 04:41:49.407580 47894079450816 run_docker-2.py:193] E0817 20:41:49.119690 140014160611136 hhblits.py:141] - 19:20:25.909 WARNING: maximum number of residues 32763 exceeded in sequence tr|A0A091T648|A0A091T648_PHALP Titin (Fragment) OS=Phaethon lepturus GN=N335_02550 PE=4 SV=1

Anyone knows what it is about?

**Comment by rainwala** [comment]
I have the same problem.

**Comment by ValZapod** [comment]
> Titin

This is from ChimeraX twitter? They did not use the full sequence. They cut in pieces.

**Comment by Augustin-Zidek** [comment]
Duplicate of https://github.com/deepmind/alphafold/issues/172. 

TL;DR: AlphaFold can fold at most 2,700-residue proteins and the sequence you are trying to run with is 32k residues. The best workaround is to split the sequence in shorter ones and fold those, then merge.

**Comment by ValZapod** [comment]
> AlphaFold can fold at most 2,700-residue proteins and the sequence you are trying to run with is 32k residues. The best workaround is to split the sequence in shorter ones

Actually what Alphafold DB does it splits into highly overlapping segments of 1400aa length overlapping fragments (residues 1–1400, residues 201–1600, etc).

**Comment by nanomolar** [comment]
I got the same problem for a complex sequence with 200 residues. Could anyone provide answers?

**Comment by noah-c-noah-do** [comment]
> I got the same problem for a complex sequence with 200 residues. Could anyone provide answers?

I also got this issue on a multimer run with sequences of 200-400. I believe the issue is that Titin proteins are being found as templates/homologs b and their size is causing the issue

**Comment by cedarice** [comment]
I have the same problem, too!

**Comment by tcoates5** [comment]
I am seeing a similar problem

**Comment by javierbq** [comment]
Same problem
